app/script/localization/translations/webapp-tr.js

```diff
@@ -102,146 +102,146 @@ z.string.tr.callJoin = 'Katıl';
-z.string.tr.modal_delete_account_headline = 'Hesabı Sil';
-z.string.tr.modal_delete_account_message = 'Size kısa mesaj veya e-posta aracılığıyla bir mesaj göndereceğiz. Mesajdaki bağlantıyı takip ederek hesabınızı kalıcı olarak silebilirsiniz.';
-z.string.tr.modal_delete_button = 'Sil';
-z.string.tr.modal_delete_headline = 'Sadece benim için sil?';
-z.string.tr.modal_delete_message = 'Bu işlem geri alınamaz.';
-z.string.tr.modal_delete_everyone_button = 'Sil';
-z.string.tr.modal_delete_everyone_headline = 'Herkes için sil?';
-z.string.tr.modal_delete_everyone_message = 'Bu işlem geri alınamaz.';
-z.string.tr.modal_too_long_headline = 'Mesaj çok uzun';
-z.string.tr.modal_too_long_message = 'En fazla {{number}} karakterlik mesajlar gönderebilirsiniz.';
-z.string.tr.modal_leave_conversation_headline = '"{{user}}" görüşmesinden ayrılınsın mı?';
-z.string.tr.modal_leave_conversation_message = 'Katılımcılar bilgilendirilecek ve görüşme listenizden silinecek.';
-z.string.tr.modal_leave_conversation_button = 'Ayrıl';
-z.string.tr.modal_logout_headline = 'Veriyi Temizle?';
-z.string.tr.modal_logout_message = 'Bu cihazdaki tüm kişisel bilgileriniz ve görüşme kayıtlarınızı silin.';
-z.string.tr.modal_logout_button = 'Çıkış yap';
-z.string.tr.modal_new_device_headline = '{{user}} yeni bir cihaz kullanmaya başladı';
-z.string.tr.modal_new_device_headline_many = '{{user}}s yeni cihazlar kullanmaya başladılar';
-z.string.tr.modal_new_device_headline_you = '{{user}} yeni bir cihaz kullanmaya başladı';
-z.string.tr.modal_new_device_message = 'Hâlâ mesajlarınızı göndermek istiyor musunuz?';
-z.string.tr.modal_new_device_call_accept = 'Aramayı kabul et';
-z.string.tr.modal_new_device_call_anyway = 'Yine de ara';
-z.string.tr.modal_new_device_call_incoming = 'Hala aramayı kabul etmek istiyor musunuz?';
-z.string.tr.modal_new_device_call_outgoing = 'Hala aramayı istiyor musunuz?';
-z.string.tr.modal_new_device_show_device = 'cihazı göster';
-z.string.tr.modal_new_device_send_anyway = 'yine de gönder';
-z.string.tr.modal_not_connected_headline = 'Hiç kimseye konuşmaya katılmadı';
-z.string.tr.modal_not_connected_message_one = '{{name}} sohbetlere eklenmek istemiyor.';
-z.string.tr.modal_not_connected_message_many = 'Seçtiğin kişilerden biri sohbetlere eklenmek istemiyor.';
-z.string.tr.modal_remove_device_button = 'Cihazı kaldır';
-z.string.tr.modal_remove_device_headline = '"{{device}}" cihazını kaldır';
-z.string.tr.modal_remove_device_message = 'Cihazı kaldırmak için şifreniz gereklidir.';
-z.string.tr.modal_session_reset_headline = 'Oturum sıfırlandı';
-z.string.tr.modal_session_reset_message_1 = 'Eğer sorun çözülmemişse,';
-z.string.tr.modal_session_reset_message_link = 'iletişime geç';
-z.string.tr.modal_session_reset_message_2 = 'biz.';
-z.string.tr.modal_too_many_members_headline = 'Dolup taşmış';
-z.string.tr.modal_too_many_members_message = 'Görüşmeye en fazla {{number1}} kişi katılabilir. Burada {{number2}} kişi için daha boş yer var.';
```

Comment:
how come that want updated yesterday?

Reply:
You tell me. Main reason for the PR was the fix of all the broken German translations. I guess other languages have the same problem, but I cannot tell.